crates/rooch-rpc-api/src/jsonrpc_types/state_view.rs

```diff
@@ -102,9 +111,20 @@ impl From<Op<State>> for OpView<StateView> {
     }
 }
 
+impl From<OpView<StateView>> for Op<State> {
+    fn from(op: OpView<StateView>) -> Self {
+        match op {
+            OpView::New(data) => Self::New(data.into()),
+            OpView::Modify(data) => Self::Modify(data.into()),
+            OpView::Delete => Self::Delete,
+        }
+    }
+}
+
 #[derive(Clone, Debug, Serialize, Deserialize, JsonSchema)]
 pub struct TableChangeView {
     pub entries: BTreeMap<BytesView, OpView<StateView>>,
+    pub size_increment: i64,
```

Comment:
Maybe this new field breaks the js sdk.

Reply:
Not this change breaks the SDK test. But the typescript `TableChangeView` also needs to be updated.